### DISCUSSION TITLE
How to enable multithread build when installing packages?

### DISCUSSION THREAD

**Discussion by zjnyly** [microsoft/vcpkg#28490] [Q&A] [answered]
I'm building OpenCV on 48 core CPU, but still very slow. There is no progress bar during building which makes me doubt whether the program is still alive.

I checked the resource usage of `vcpkg`, only one thread is created and the CPU usage is close to zero. 

How can I speed up the building process like `make -j8` ? Thanks!

![image](https://user-images.githubusercontent.com/56783525/209096814-227c94f3-545a-463c-aa19-5e0287bba0be.png)

**Comment by Osyotr** [comment, marked as answer]
Vcpkg does not consume resources because it just waits for cmake process to finish.
CMake also does not consume resources because it waits for Ninja.
Ninja uses all your cores by default (adjustable with `VCPKG_MAX_CONCURRENCY` env variable).
Also, `ld` does not use parallelism.
